src/ingestion/zengrc.py

```diff
@@ -0,0 +1,67 @@
+import requests
+import base64
+import datetime
+import json
+import os
+
+from runners.helpers import db, log
+
+ZENGRC_ID = os.environ.get('ZENGRC_ID')
+ZENGRC_SECRET = os.environ.get('ZENGRC_SECRET')
+ZENGRC_URL = os.environ.get('ZENGRC_URL')
+ZENGRC_TABLE = os.environ.get('ZENGRC_TABLE')
+TIMESTAMP = str(datetime.datetime.utcnow())
+
+
+def gen_headers(id, secret):
```

Comment:
please use the auth param like we do in auth.py — https://github.com/snowflakedb/SnowAlert/blob/master/src/runners/helpers/auth.py#L42

Reply:
done